src/deck-options.md

```diff
@@ -2,262 +2,223 @@
 
 <!-- toc -->
 
-Deck options primarily control the way Anki schedules cards. It is recommended
+Deck options primarily control the way Anki schedules cards. If you're a new user, it's recommended
 that you spend a few weeks with the defaults to get a feel for how Anki works
-before you start adjusting options. Please make sure you understand the options
+before you start adjusting the options. Please make sure you understand the options
 before changing them, as mistakes could reduce Anki's effectiveness.
 
-Deck options are accessed by:
+If you're on the desktop app, do any of the following to access deck options:
 
-- Clicking the gear icon on the `Decks` screen.
-- Selecting a deck on the `Decks` screen, and then clicking `Options`
+- Click the gear icon on the Decks screen.
+- Select a deck on the Decks screen, and then click "Options"
   at the bottom of the screen.
-- Clicking on `More` > `Options` while in review mode.
-- Pressing `o` while in review mode.
+- Click "More" > "Options" while in review mode.
+- Press <kbd>o</kbd> while in review mode.
```

Comment:
```suggestion
- Click **More** > **Options** while studying.
- Press <kbd>o</kbd> while studying.
```

Reply:
Goggle style is wrapping the whole part with bold tags. In Markdown, this would be `**More > Options**`. I think this one is what Microsoft does but it's busy work and I don't like it. Should I proceed with Google style?

Reply:
You proposed "bold," so I changed it to bold when making my other edits. I have no particular allegiance to one style over another. If there's something specific you want settled, that's a question for #253 -- and it should be documented.

Reply:
I linked to how Google handles UI elements.

Reply:
This doesn't appear resolved to me.

Reply:
I've unresolved this as you asked. This highlighting issue only came up later, so I wasn't exactly planning to do this here.